src/data/mod.rs

```diff
@@ -173,18 +174,27 @@ impl Qualifiers {
     pub const NONE: Qualifiers = Qualifiers {
         c_const: false,
         volatile: false,
+        inline: false,
     };
     pub const VOLATILE: Qualifiers = Qualifiers {
         c_const: false,
         volatile: true,
+        inline: false,
     };
     pub const CONST: Qualifiers = Qualifiers {
         c_const: true,
         volatile: false,
+        inline: false,
     };
     pub const CONST_VOLATILE: Qualifiers = Qualifiers {
         c_const: true,
         volatile: true,
+        inline: false,
+    };
+    pub const INLINE: Qualifiers = Qualifiers {
+        c_const: false,
+        volatile: false,
+        inline: true,
```

Comment:
You don't need to add this unless you use it somewhere, the other `const` qualifiers are just helpers for unit tests mostly.